### PR TITLE
Try to simplify drone config

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Just me playing around with the config and (hopefully) making things simpler and faster

### CODE REVIEW

**Comment by pascalwengerter** [comment]
@micbar and @wkloucek  could you give this a good look? Trying to get into the CI config stuff a bit, managed to reduce complexity in the build pipeline (from my POV) by a bit and improved runtime, esp. on releases, too:

**Before (examples from today):**
Normal run: ~4min https://drone.owncloud.com/owncloud/owncloud-design-system/3806
Release: 13min https://drone.owncloud.com/owncloud/owncloud-design-system/3811

**After (my changes):**
Normal run: Also ~4min https://drone.owncloud.com/owncloud/owncloud-design-system/3818
Release: 6min https://drone.owncloud.com/owncloud/owncloud-design-system/3819/1/9

File: .drone.star [hunk 1]
```diff
@@ -26,192 +22,161 @@ def testing(ctx):
...
+            'commands': [
+                'git clone https://github.com/%s.git .' % (repo_slug),
+                'git checkout $DRONE_COMMIT',
+            ]
         },
-        "clone": {
-            # The default drone clone step "internally auto-merges" the PR branch to master.
-            # That confuses SonarCloud. The analysis report does not come from the actual
-            # real commit at the tip of the branch.
-            "disable": True,
```
Comment (micbar) [inline .drone.star:41]: This needs to be kept. The reason behind it is sonarcloud. Normally, drone tries so "pre-merge" the pr branch into master. For the unit tests and the upload of the coverage, we need to disable the default drone "clone and merge" routine and do the clone manually.

Comment (pascalwengerter) [reply .drone.star:41]: Should just have "changed line numbers" and can be found lower in the file :)

**Comment by pascalwengerter** [comment]
Closing this one in favor of #1627 which has the same changes included

**Pull request closed without merging**